### PR TITLE
Feature/support runtime env casting

### LINKED ISSUE #910: Adopt runtime environment parameter / argument feature
**Is your feature request related to a problem? Please describe.**

When starting session, environment parameters can be set to the instance for supporting images. Let's adopt runtime environment parameter / argument feature to GUI too.

**Describe the solution you'd like**

Runtime environment parameter / argument feature. Wsproxy need to be patched too.

### PR DESCRIPTION
**Pull request by inureyes**
This PR resolves #910

### CODE REVIEW

**Comment by inureyes** [comment]
TODO: environment parsing example. I'll add it to file browser on vfolder code.

**Review by lizable — APPROVED** [review]
LGTM

File: src/wsproxy/lib/backend.ai-ws-appproxy.js [hunk 1]
```diff
@@ -54,21 +54,27 @@ module.exports = (proxy = class Proxy extends ai.backend.Client {
...
     this.port = port;
-    this._conn(sessionName, app);
+    this._conn(sessionName, app, envs, args);
     this.tcpServer.listen(this.port, this.ip);
   }
 
-  _conn(sessionName, app) {
+  _conn(sessionName, app, envs, args) {
+    if (typeof envs === 'object' && Object.keys(envs).length > 0) {
+      app = app + '&envs='+ JSON.stringify(envs);
```
Comment (adrysn) [inline src/wsproxy/lib/backend.ai-ws-appproxy.js]: I think this should be url-encoded for safety. If url is encoded here, the receiver (`manager.js`) should also url-decode before parsing JSON string. For example, I'm not sure the following `envs` will be delivered properly as a querystring.

```javascript
'&envs='+JSON.stringify({myenv: 'ab&cd=e'})
> "&envs={"myenv":"ab&cd=e"}"
```

**Review by adrysn — CHANGES_REQUESTED** [review]
I think simple `JSON.stringify(envs)` is not safe to be sin query string. Perhaps, we can wrap it with `encodeURI`.

**Review by adrysn — APPROVED** [review]

**Pull request merged**